articles/user-guide/host-programs.md

```diff
@@ -315,12 +315,22 @@ A Python host program is constructed as follows:
 
     With the `qsharp` module imported, you can also import callables directly from the Q# library namespaces.
 
-3. Among any other Python code, you can now call those callables on specific target machines, and assign their returns to variables (if they return a value) for further use.
+3. Alongside regular Python code, you can now run those callables on specific target machines (see next section), and assign their return values to variables for further use:
```

Comment:
```suggestion
3. Alongside regular Python code, you can now run those callables on [specific target machines](#specifying-target-machines), and assign their return values to variables for further use:
```

Reply:
I know the next section is *literally* the next line, but we try to avoid directional wording, like "above", "below", "bottom of page", etc. "Previous" and "following" are okay, as "in the previous example...". It has to do with accessibility guidelines and screen readers.

Reply:
Thanks, I will keep this in mind!